assets/app/widgets/suse-lazy-spinner/suse-lazy-spinner.directive.js

```diff
@@ -0,0 +1,60 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('crowbarWidgets')
+        .directive('suseLazySpinner', suseLazySpinner)
+        .constant('SUSELS_DEFAULT_DELAY', 2000);
```

Comment:
I was thinking about this too. Changed now.
As for a separate `constants` file, I think we can keep it inside the directive file as long as it's only one constant and it's used only by this directive. It can be refactored when the situation changes.